flocker/acceptance/test_api.py

```diff
@@ -432,3 +432,26 @@ def not_exists():
             return deleted
         created.addCallback(delete_dataset)
         return created
+
+    def test_dataset_grow_(self):
+        """
+        The maximum size of a dataset can be increased.
+
+        A not quite end-to-end test that the size of a dataset can be increased.
+
+        We can't issue the resize request from the API until
+        https://clusterhq.atlassian.net/browse/FLOC-1599 is resolved.
+
+        So manipulate the control service configuration file directly and
```

Comment:
I suggest just pushing the acceptance test into a separate issue that is blocked on 1599, no point wasting time on doing things the hard way when we can do the easier (and more realistic!) way soon.